src/pygenesis/profile.py

```diff
@@ -0,0 +1,66 @@
+"""Module provides wrapper for Profile GENESIS REST-API functions."""
+
+from pygenesis.config import (
+    _write_config,
+    get_config_path_from_settings,
+    load_config,
+)
+from pygenesis.http_helper import get_data_from_endpoint
+
+
+def password(new_password: str) -> str:
```

Comment:
Name sollte etwas besser zur Funktion passen, `password` heißt für mich erstmal primär Passwort abfragen. Dass sich hier eine Setter-Methode dahinter verbirgt wäre mir nicht klar, daher würde ich vorschlagen, den Namen in `set_password` oder `change_pw` zu ändern, von der Idee her.